lib/api/completeMultipartUpload.js

```diff
@@ -433,6 +434,11 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                     metaStoreParams.versioning = options.versioning;
                     metaStoreParams.isNull = options.isNull;
                     metaStoreParams.nullVersionId = options.nullVersionId;
+                    const replicationInfo = getReplicationInfo(objectKey,
```

Comment:
Thank you. I think it would be appropriate to instead move this to the previous waterfall function.